### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by json-born**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,27 @@
...
+- A game can be created with 2 players
+- Each players score starts at 0
+- Each player can score a point
+- The score of a player in the game increments the following way: 0 - 15 - 30 - 40
+- The score resets to 0-0 when one player reaches 40.
+- Deuce can be reached if players score is tied at 40-40
+- Advantage point will be awarded if a player scores at 40-40
+- The game returns to deuce if one player is at advantage and the other scores a point
+- The score resets to 0-0 if one player scores twice in a row at Deuce
+
```
Comment (digitalgears) [inline README.md:23]:
```suggestion
- Does anyone actually read this far?
```

**Review by pstonham — APPROVED** [review]

**Review by jeffwad — CHANGES_REQUESTED** [review]
Mwah ha ha! I still has access...

**Comment by jeffwad** [comment]
Feel free to remove my account/email address from this repo.

File: README.md [hunk 2]
```diff
@@ -1,18 +1,27 @@
...
 
+## The Task
+
+Your solution should implement the following features:
+
+- A game can be created with 2 players
+- Each players score starts at 0
+- Each player can score a point
+- The score of a player in the game increments the following way: 0 - 15 - 30 - 40
+- The score resets to 0-0 when one player reaches 40.
```
Comment (thardyman) [inline README.md]: This is a confusing way of saying someone wins the game.  
```suggestion
- A player wins the game if they score after reaching 40, and the match is not at Duece.
```

File: README.md [hunk 3]
```diff
@@ -1,18 +1,27 @@
...
+- Deuce can be reached if players score is tied at 40-40
+- Advantage point will be awarded if a player scores at 40-40
+- The game returns to deuce if one player is at advantage and the other scores a point
+- The score resets to 0-0 if one player scores twice in a row at Deuce
```
Comment (thardyman) [inline README.md]: Again, the scenario is a win...
```suggestion
- If one player scores twice in a row at Deuce then they win the game
```

**Review by thardyman — APPROVED** [review]

**Comment by thardyman** [comment]
> Feel free to remove my account/email address from this repo.

Lol, thanks for letting us know!

**Pull request merged**